pallets/ddc-payouts/src/lib.rs

```diff
@@ -235,8 +254,9 @@ pub mod pallet {
 		NotInitialized,
```

Comment:
As we changed `ValueQuery` to `OptionQuery`, maybe we should remove the `NotInitialized` status now ?

Reply:
we can merge several status together as part of optimisations - we'll deal with it as well once the time comes

Reply:
ok, let's leave it like this for now